src/main/java/net/william278/velocitab/commands/VelocitabCommand.java

```diff
@@ -1,24 +1,24 @@
 package net.william278.velocitab.commands;
 
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.velocitypowered.api.command.BrigadierCommand;
 import com.velocitypowered.api.command.CommandSource;
-import com.velocitypowered.api.command.SimpleCommand;
 import net.kyori.adventure.text.Component;
 import net.kyori.adventure.text.format.TextColor;
 import net.william278.desertwell.AboutMenu;
 import net.william278.desertwell.Version;
 import net.william278.velocitab.Velocitab;
 
-import java.util.List;
-
-public class VelocitabCommand implements SimpleCommand {
+public final class VelocitabCommand {
     private final AboutMenu aboutMenu;
     private final Velocitab plugin;
 
-    public VelocitabCommand(Velocitab plugin) {
+    public VelocitabCommand(final Velocitab plugin) {
```

Comment:
```suggestion
    public VelocitabCommand(@NotNull Velocitab plugin) {
```
minor nit: Actually something I missed in the last PR, but yeah should probably be annotated NotNull and a final is unnecessary here I feel